src/components/Editors/View/Query.tsx

```diff
@@ -13,7 +13,7 @@ export default function Query() {
   const query = useStore((state) => state.view.query)
   const setQuery = useStore((state) => state.setQuery)
   const viewError = useStore((state) => state.viewError)
-  const aceEditor = React.useRef<AceEditor>(null);
+  const aceEditor = React.useRef<AceEditor>(null)
```

Comment:
this is there to make it able us to refer to the editor instance and be able to use its api to insert new text